lib/src/components/_stacks-tables.less

```diff
@@ -197,17 +151,172 @@
     }
 }
 
-//  $$  Footer: Large Row
-//  ------------------------------------------------------------------------
-.s-table--totals {
-    th,
-    td {
-        padding-top: @su12;
-        font-size: @fs-subheading;
-        font-weight: bold;
 
-        &:hover {
-            background-color: transparent;
+//  ============================================================================
+//  $ LAYOUT VARIATIONS
+//  ----------------------------------------------------------------------------
+.s-table {
+    //  $$  HOVER
+    //  ------------------------------------------------------------------------
+    &.s-table__hover tbody tr:hover {
+        background-color: @black-025;
+    }
+
+    //  $$  SPACING
+    //  ------------------------------------------------------------------------
+    //  --  SMALL
+    &.s-table__sm {
+        th,
+        td {
+            padding: @su4;
         }
     }
+    //  --  MEDIUM
+    &.s-table__md {
+        th,
+        td {
+            padding: @su12;
+        }
+    }
+    //  --  LARGE
+    &.s-table__lg {
+        th,
+        td {
+            padding: @su16;
+        }
+    }
+
+    //  $$  NO BORDERS
+    //  ------------------------------------------------------------------------
+    //  --  X & Y Axis
+    &.s-table__b0 {
+        th,
+        td,
+        tr:last-of-type th,
+        tr:last-of-type td {
+            border-color: transparent;
+        }
+
+        //  This makes the border transparent, so we need to use whitespace
+        //  to achieve the same effect a 2px gray border achieves.
+        tbody + tbody {
+            border-color: transparent;
+            border-width: @su12;
+        }
+    }
+    //  --  X-AXIS
+    //      Clear out the y-axis colors
```

Comment:
Is this comment correct?

Reply:
These class names seem backwards to me. If I'm applying `bx0`, I expect there to be no horizontal borders, but the documentation has the opposite?

Reply:
This was modified from @TedGoas's original class name of `s-table__borderlessx`, which I felt was a bit verbose. I read `.s-table__bx0` as x-axis borders have a 0 width. But if that's not clear, then we should change it. Would `s_table__bx-transparent` be clearer? It's still verbose, but it at least matches some of our border naming conventions.